holoviews/plotting/bokeh/element.py

```diff
@@ -561,7 +561,8 @@ def _plot_properties(self, key, element):
         if self.border is not None:
             for p in ['left', 'right', 'top', 'bottom']:
                 plot_props['min_border_'+p] = self.border
-        lod = dict(self.param.defaults().get('lod', {}), **self.lod)
+        defaults = {k: v.default for k, v in self.param.objects().items()}
+        lod = dict(defaults.get('lod', {}), **self.lod)
```

Comment:
```suggestion
        lod = dict(self.params["lod"].default, **self.lod) if "lod" in self.param else self.lod
```

I don't think a complete dict lookup of `param.objects()` is needed here.